pkg/restore/crd.go

```diff
@@ -32,7 +32,8 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
-func RestoreCrd(ctx context.Context, k8sclientset *k8sutil.Clientsets, operatorNamespace, clusterNamespace string, args []string) {
+func RestoreCrd(ctx context.Context, k8sclientset *k8sutil.Clientsets, operatorNamespace, clusterNamespace string,
+	groupNameOptional, versionResourceOptional *string, args []string) {
```

Comment:
Let's just require these names, it seems simpler this way.
```suggestion
	groupName, versionResource string, args []string) {
```

Reply:
done